src/sentry/incidents/logic.py

```diff
@@ -808,6 +810,22 @@ def subscribe_projects_to_alert_rule(alert_rule, projects):
     )
 
 
+def enable_alert_rule(alert_rule):
+    if alert_rule.status != AlertRuleStatus.DISABLED.value:
+        return
+    with transaction.atomic():
+        alert_rule.update(status=AlertRuleStatus.PENDING.value)
+        bulk_enable_snuba_subscriptions(alert_rule.snuba_query.subscriptions.all())
+
+
+def disable_alert_rule(alert_rule):
+    if alert_rule.status == AlertRuleStatus.DISABLED.value:
```

Comment:
Wonder if this should just be `alert_rule.status != AlertRuleStatus.ENABLED.value`. Not sure if we want to be able to disable a snapshot, since then that'd end up allowing it to be re-enabled.

Reply:
Just checking on whether you're changing this or if there's a reason to keep it as is?